src/main.rs

```diff
@@ -88,8 +88,8 @@ struct Opts {
     connect_under_reset: bool,
 
     /// Enable more verbose logging.
-    #[structopt(short, long)]
-    verbose: bool,
+    #[structopt(short, long, multiple(true))]
+    _verbose: bool,
```

Comment:
This will put the number of `-v` flags into the `verbose` field:

```suggestion
    #[structopt(short, long, parse(from_occurrences))]
    verbose: u32,
```

Reply:
Oh wow. I've tried that, but I didn't work because I misspelled "occurrences"...